sites/partners/styles/overrides.scss

```diff
@@ -25,3 +25,10 @@
   // Table row hover background color
   --ag-row-hover-color: #f2f2f2;
 }
+
+Button.button.is-primary {
```

Comment:
Does it work if we unnest this? 

```suggestion
Button.button.is-primary:hover { 
```

Similar comment for the other file.

Reply:
Yes, that works